### PR TITLE
Release notes for 1.1.1

### PR DESCRIPTION
**Pull request by duderino**

### CODE REVIEW

File: content/about/notes/1.1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+Istio 1.1.1 is the first in a weekly series of patch releases for Istio 1.1.  It contains a small number of bug fixes and minor enhancements.
+
+{{< relnote_links >}}
+
+## Bug Fixes and Minor Enhancements
+
+- Configure Prometheus to monitor citadel ([Issue 12175](https://github.com/istio/istio/pull/12175))
+- Improve output of `istioctl's` verify install command ([Issue 12174](https://github.com/istio/istio/pull/12174))
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: Improve the output of the `istioctl verify install` command (and make that a link)

File: content/about/notes/1.1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-22
+icon: notes
+---
+
+Istio 1.1.1 is the first in a weekly series of patch releases for Istio 1.1.  It contains a small number of bug fixes and minor enhancements.
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: I don't think we should commit to this being weekly.

Also, can we use the same text as in previous patch releases? which is:

This release includes security vulnerability fixes and improvements to robustness.
This release note describes what's different between Istio 1.1.1 and Istio 1.0.

File: content/about/notes/1.1.1/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+{{< relnote_links >}}
+
+## Bug fixes and minor enhancements
+
+- Configure Prometheus to monitor Citadel ([Issue 12175](https://github.com/istio/istio/pull/12175))
+- Improve output of [`istioctl verify-install`](/docs/reference/commands/istioctl/#istioctl-experimental-verify-install) command ([Issue 12174](https://github.com/istio/istio/pull/12174))
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: Actually, this should be istioctl experimental verify-install

File: content/about/notes/1.1.1/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Reduce log level for missing service account for SPIFFE URI ([Issue 12108](https://github.com/istio/istio/issues/12108))
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: ...missing service account messages for a SPIFFEE URI

File: content/about/notes/1.1.1/index.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- Fix broken path on the opt-in SDS feature's Unix domain socket ([Issue 12688](https://github.com/istio/istio/pull/12688))
+- Fix Envoy tracing: If parent span is propagated with empty string, it causes the next child span to not be created ([Envoy Issue 6263](https://github.com/envoyproxy/envoy/pull/6263))
```
Comment (geeknoid) [inline content/about/notes/1.1.1/index.md]: Fix Envoy tracing that was preventing a child span from being created if the parent span was propagated with an empty string.

**Comment by geeknoid** [comment]
This change needs to be applied to the release-1.1 branch instead.

And please don't forget to include the two line blog post announcing the release. Just clone the existing patch release announcements.

File: content/about/notes/1.1.1/index.md (hunk 4 above)
Comment (duderino) [reply content/about/notes/1.1.1/index.md]: SPIFFE not SPIFFEE

**Comment by duderino** [comment]
Replacing with https://github.com/istio/istio.io/pull/3845

**Pull request closed without merging**